src/lifetracker/logic/CommandLineResult.java

```diff
@@ -4,9 +4,14 @@
 import java.util.List;
 
 public class CommandLineResult implements ExecuteResult {
+	
+    public enum CommandType {
```

Comment:
Define the enum in the interface.

If you put it in the implementation, then the implementation depends on the abstraction, and not the other way round, which defeats the purpose of the abstraction in the first place.